File: Sources/gestion-culturelle-front/project/src/app/model/animationDto.ts
import { User } from "./user";

export class AnimationDto {
    id: number;
    label: string;
    type: string;
    prix: number;
    nbreSpectateursPrevus: number;
    animateur:User;
    

    constructor(id?: number,
        label?: string,
        type?: string,
        prix? : number,
        nbreSpectateursPrevus?: number,
        animateur?:User ){
        this.id = id;
        this.label=label;
        this.type = type;
        this.prix = prix;
        this.nbreSpectateursPrevus = nbreSpectateursPrevus;
        this.animateur = animateur;
       
    }
}

File: Sources/gestion-culturelle-back/src/main/java/com/afpa/cda/dto/AnimationDto.java
package com.afpa.cda.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@Builder
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@ToString
public class AnimationDto {
	private int id;
	private String label;
	private String type;
	private int prix;
	private int nbreSpectateursPrevus;
	private UserDto animateur;
}
